src/applications/edu-benefits/sass/edu-benefits.scss

```diff
@@ -164,3 +164,42 @@
     margin-right: 0;
   }
 }
+
+.stem-eligibility-page {
+  max-width: 100%;
+  width: 100%;
+}
+
+
+.stem-eligibility {
+  background-color: #f1f1f1;
```

Comment:
This can also be accomploished with the design system:

- `vads-u-background-color--gray-lightest` as a class
- `$color-gray-lightest` as a SASS variable.

https://design.va.gov/utilities/background-color

Reply:
changed